test/test.ts

```diff
@@ -1,16 +1,34 @@
-import { VhdlLinter } from '../lib/vhdl-linter';
+import { OIDiagnostic, VhdlLinter } from '../lib/vhdl-linter';
```

Comment:
maybe `OIDiagnostic` should be renamed to `IDiagnostic` to match our naming scheme

Reply:
I think it is calles OIDiagnostic because the position is based on OI :D